### PR TITLE
Add node exporter

### PR DESCRIPTION
**Pull request by darora**
- fix aws cli arch

### CODE REVIEW

File: ansible/tasks/setup-supabase-internal.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - name: AWS CLI
   get_url:
-    url: "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-{{ aws_cli_release }}.zip"
+    url: "https://awscli.amazonaws.com/awscli-exe-linux-aarch64-{{ aws_cli_release }}.zip"
```
Comment (inian) [inline ansible/tasks/setup-supabase-internal.yml:9]: This should be dynamically configured [like this](https://github.com/supabase/postgres/blob/d26bec8d840e363f21fa9fe3c1acbe33f2363787/ansible/tasks/setup-misc.yml#L16) right?

File: ansible/tasks/node-exporter.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+- name: UFW - Allow connections to node exporter ports
+  ufw:
+    rule: allow
+    port: "9100"
+
+- name: Node Exporter - download binary archive
+  get_url:
+    url: "https://github.com/prometheus/node_exporter/releases/download/v{{ node_exporter_release }}/node_exporter-{{ node_exporter_release }}.linux-arm64.tar.gz"
```
Comment (inian) [inline ansible/tasks/node-exporter.yml:8]: detect arch here too?

File: ansible/tasks/postgres-exporter.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    owner: root
+    mode: '0700'
+  become: yes
+  with_items:
+    - /opt/postgres_exporter
+    - /etc/systemd/system/postgres_exporter.service.d
+
+- name: download postgres exporter
+  get_url:
+    url: "https://github.com/prometheus-community/postgres_exporter/releases/download/v{{ postgres_exporter_release }}/postgres_exporter-{{ postgres_exporter_release }}.linux-arm64.tar.gz"
```
Comment (inian) [inline ansible/tasks/postgres-exporter.yml:19]: detect arch here too?

File: ansible/tasks/node-exporter.yml (hunk 2 above)
Comment (darora) [reply ansible/tasks/node-exporter.yml]: done

File: ansible/tasks/setup-supabase-internal.yml (hunk 1 above)
Comment (darora) [reply ansible/tasks/setup-supabase-internal.yml]: done

File: ansible/tasks/postgres-exporter.yml (hunk 3 above)
Comment (darora) [reply ansible/tasks/postgres-exporter.yml]: done

**Review by inian — APPROVED** [review]

**Pull request merged**